app/resources/api/v2/labware_resource.rb

```diff
@@ -2,7 +2,18 @@
 
 module Api
   module V2
-    # LabwareResource
+    # @todo This documentation does not yet include a detailed description of what this resource represents.
+    # @todo This documentation does not yet include detailed descriptions for relationships, attributes and filters.
+    # @todo This documentation does not yet include any example usage of the API via cURL or similar.
+    #
+    # @note This resource is immutable: its endpoint will not accept `POST`, `PATCH`, or `DELETE` requests.
```

Comment:
Is this immutable?

Reply:
Quite right.  I will remove this line from the YARD documentation.